extensions/rocksdb-repos/RocksDatabase.h

```diff
@@ -84,6 +84,7 @@ class RocksDatabase {
   };
 
   RocksDatabase(const rocksdb::Options& options, const std::string& name, Mode mode = Mode::ReadWrite);
+  virtual ~RocksDatabase() = default;
```

Comment:
Since there are no derived classes of `RocksDatabase`, I would rather make all members non-virtual.

Reply:
Updated in 267cda83a77a5a6d05f9df714d0c923814351826